tmt-app/src/Components/Quiz.jsx

```diff
@@ -31,21 +34,21 @@ function Quiz() {
 
 function Text({ step }) {
   const textContent = {
-    1: {
+    classes: {
       title1: "Let's order some boba! Pick your base(s).",
       title2: "What university classes sound interesting to you?",
       subtitle: "Select more than 3.",
       bobaBot: "5 is the maximum.",
     },
-    2: {
+    skills: {
       title1: "Pick your toppings and add-ons.",
       title2: "What skills would you like to use in a career?",
       subtitle: "Select more than one.",
       bobaBot: "6 is the maximum.",
     },
   };
 
-  const content = textContent[step] || textContent["1"];
+  const content = textContent[step] || textContent["classes"];
```

Comment:
Can simplify these with just .classes instead of a string key if it's a constant